src/middlewared/middlewared/plugins/virt/instance.py

```diff
@@ -36,9 +36,10 @@ async def query(self, filters, options):
         """
         Query all instances with `query-filters` and `query-options`.
         """
-        config = await self.middleware.call('virt.global.config')
-        if config['state'] != 'INITIALIZED':
-            return []
+        if not options.get('extra').get('skip_state'):
```

Comment:
`options.get('extra').get()` seems kinda weird..iirc extra should always be there now

Reply:
Indeed, thanks